src/cipher/mod.rs

```diff
@@ -0,0 +1,319 @@
+use dusk_jubjub::AffinePoint;
+use dusk_plonk::prelude::BlsScalar;
```

Comment:
Nit: as mentioned in a previous PR, idiomatically preludes should be used mostly as `prelude::*`, in case of PLONK when we need the constraint system. If you just need `BlsScalar` you can use `dusk_plonk::dusk-bls12_381::Scalar as BlsScalar`.
Since we're accessing to `BlsScalar` very often, as shortcut we might think to re-export them directly from `dusk_plonk` crates, what do you think? Often it's the only thing we actually need.